config/kub/MenilSaintMichel.json

```diff
@@ -31,13 +31,13 @@
         "stages": [
             {
                 "name":"",
-                "file": "instances/np_{{parameters.nb_tasks.tasks.value}}/logs/timers.json",
+                "filepath": "instances/np_{{parameters.nb_tasks.tasks.value}}/logs/timers.json",
                 "format": "json",
                 "variables_path":"*"
             },
             {
                 "name":"",
-                "file": "logs/timers.json",
+                "filepath": "logs/timers.json",
                 "format": "json",
                 "variables_path":"*"
             }
```

Comment:
is it needed to convey the fact that you may have file without path ?
file was ok to me, it refers to the location of the file
filepath is needed perhaps it is important in the workflow that knowing explicitely the fact that this is a file with a path